### PR TITLE
Make <eor> upper case

### PR DESCRIPTION
**Pull request by phl0**
As all other ADIF tages are exported upper case it would make sense to also have <EOR> instead of <eor>.

### CODE REVIEW

**Comment by magicbug** [comment]
@phl0 @AndreasK79 Just to say this was a breaking change to LoTW and has been reverted. Can be resubmitted once further testing carried out

File: application/views/lotw_views/adif_views/adif_export.php [hunk 1]
```diff
@@ -10,7 +10,7 @@
 <CERT_UID:1>1
 <CERTIFICATE:<?php echo strlen(trim($cert2)) + 1; ?>><?php echo trim($cert2); ?>
 
-<eor>
+<EOR
```
Comment (phl0) [inline application/views/lotw_views/adif_views/adif_export.php:13]: This is causing the LotW issues.

**Comment by phl0** [comment]
Found the usse. Was an awkward mistake by me having lost a closing tag on one EOR in the LotW section. See https://github.com/magicbug/Cloudlog/pull/1603/files#r973873612. I fixed it and made a new PR https://github.com/magicbug/Cloudlog/pull/1607. This has been tested and uploads to LotW fine. Other tests welcome.

**Comment by phl0** [comment]
So this is for the people that need to fix their logs and have would to do so the SQL way. Find the latest QSO uploaded successfully from the LotW online view. Then lookup the ID of the QSO inside Cloudlog (In my case it was F6BIR/P):

Screenshot from 2022-09-19 07-56-06

With this you can update your cloudlog table using this SQL command:

`UPDATE `TABLE_HRD_CONTACTS_V01` SET COL_LOTW_QSLSDATE = NULL, COL_LOTW_QSL_SENT = 'N', COL_LOTW_STATUS = NULL WHERE COL_PRIMARY_KEY > 34647 `

**Comment by AndreasK79** [comment]
> @phl0 @AndreasK79 Just to say this was a breaking change to LoTW and has been reverted. Can be resubmitted once further testing carried out

Sorry about that. I didn't spot that problem. Testing is always the key!

**Pull request merged**